packages/govern-server/package.json

```
@@ -4,7 +4,8 @@
   "license": "GPL-3.0",
   "main": "src/index.ts",
   "scripts": {
-    "start": "node --loader ts-node/esm.mjs  src/index.ts"
+    "start": "yarn start:containers && node --loader ts-node/esm.mjs  src/index.ts",
```

I think this should work now:

```suggestion
    "start": "yarn start:containers && ts-node src/index.ts",
```

(I started using `node --loader ts-node/esm.mjs` when I was having troubles to load the modules, but I noticed later that it was working fine with `ts-node` directly.)

---

Yup.. kept the command as it is cause I saw the change in your PR :-)